src/systems/ec/themes/default/aliases/colors-bg.yml

```diff
@@ -21,6 +21,49 @@ props:
     category: background-color
   - name: BG_COLOR_SHADE
     value: '{!COLOR_GREY_100}'
-    comment: Tertiary background colour
+    comment: Shade background colour
+    type: color
+    category: background-color
+
+  # Greys
```

Comment:
how do you know those colours are allowed as background colours?

Reply:
well it's not written, but lots of components uses grey bakgrounds